### ISSUE TITLE
agenda 0506

### ISSUE THREAD

**Issue by thin-lizard** [WeberLab-UW/LS-phd#159]
- Quick check-in with policy work 
    - [spreadsheet for data collection](https://docs.google.com/spreadsheets/d/1IG163bI3dyKgaDyMAlbm5jRuQ9xhLeCuyIotksH9nGg/edit#gid=980169151)
- PayUp study preregistration [draft](https://docs.google.com/document/d/1uDaUmome0n74WeIvNmEQnT1DN5ufcBrUGbGTsi1BhXQ/edit)
- Kelly, L. (2024). Supermarket ‘dark jobs’ and rapid grocery delivery: Transformations in labour, technology and logistics. New Media & Society, 26(3), 1227–1248. https://doi.org/10.1177/14614448231201641
    - TLDR - useful for thinking about ghost kitchen research Uses interviews with personal shoppers and 'dark store' employees, supplemented with corporate documentation & media statements, to describe the impact of e-commerce on Australian supermarkets 
-
